orchagent/portsorch.cpp

```diff
@@ -3666,6 +3672,15 @@ void PortsOrch::doLagMemberTask(Consumer &consumer)
             continue;
         }
 
+        /* Fail if a port type is not a valid type for being a LAG member port.
+         * Erase invalid entry, no need to retry in this case. */
+        if (!isValidPortTypeForLagMember(port))
```

Comment:
This specific check might not be needed, as it should be covered in the above API getPort() which will return false on failure to find a match. Was there a specific reason it was explicitly added

Reply:
getPort() does not check if this port can be a valid lag member, this check is needed.

Reply:
Sure ok .. I see. I was referring to the other overloaded call, but here we pass the alias as the parameter https://github.com/Azure/sonic-swss/blob/736f739fc7dc6593a03feb21706eafde99ae0b78/orchagent/portsorch.cpp#L668.